css/component_styles/alert.css

```diff
@@ -0,0 +1,64 @@
+/* -------- alert css -------- */
+:root {
+    --color-success: #38A169;
+    --color-warning: #DD6B20;
+    --color-error: #E53E3E;
+    --color-info: #4C95D6;
+    --color-success-bg: #C6F6D5;
+    --color-warning-bg: #FEEBC8;
+    --color-error-bg: #FED7D7;
+    --color-info-bg: #BEE3F8;
+}
+
+.alert {
+    font-family: Arial, Helvetica, sans-serif;
+    width: 30rem;
+    margin: 5px auto;
+    padding: 1rem;
+    background-color: var(--bg-color);
+    border-radius: 2px;
+    border: solid 1px transparent;
+}
+
+/* simple alert */
+.primary {
+    background-color: var(--color-info-bg);
+}
+
+.error {
+    background-color: var(--color-error-bg);
+}
+
+.success {
+    background-color: var(--color-success-bg);
+}
+
+.warning {
+    background-color: var(--color-warning-bg);
+}
+
+/* outline alert */
+.outline-primary {
+    border-color: var(--color-info);
+    color: var(--color-info);
+}
+
+.outline-error {
+    border-color: var(--color-error);
+    color: var(--color-error);
+}
+
+.outline-success {
+    border-color: var(--color-success);
+    color: var(--color-success);
+}
+
+.outline-warning {
+    border-color: var(--color-warning);
+    color: var(--color-warning);
+}
+
+#alert1,
+#alert2 {
```

Comment:
Not sure, to which element you gave this ids

I am guessing that you added this id to the alert container, if so it would be better to add the class to both the container and then use that class to style. 

It is not recommended to use Ids for styling

Reply:
changed this too ! ✔️